windowpopup.js

```diff
@@ -0,0 +1,249 @@
+    image: 'img/Desktop/Snapshoot-Portfolio3.svg',
+    description: 'Exploring the future of media in Facebook\'s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['card', 'article-card', 'mb-3', 'm-sm-auto', 'p-sm-2', 'hide-ruby'], cardImage: ['col-md-6', 'images'], cardDescription: ['col-md-6'] },
+    rowClass: { row: ['row', 'g-0', 'p-2'] },
+  },
+
+  {
+    title: 'Uber Navigation',
+    roles: { company: 'Uber', position: ['<span class="fw-bold">Uber</span>', 'Lead Developer', '2018'] },
+    image: 'img/Desktop/Snapshoot-Portfolio4.svg',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car..',
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['card', 'article-card', 'mb-3', 'm-sm-auto', 'p-sm-2', 'hide-ruby'], cardImage: ['col-md-6', 'images'], cardDescription: ['col-md-6'] },
+    rowClass: { row: ['row', 'g-0', 'd-flex', 'flex-row', 'flex-row-reverse', 'p-2'] },
+  },
+];
+
+const desktopDummyText = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea';
+
+const mobileDummyText = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.';
```

Comment:
- [ ] Please, make sure that when we click on "See project" bottom, the popup with details about the project must be the same on the mobile and desktop versions.

Reply:
@OLIPLICHE  The desktop version on Figma seems to have more text than the mobile version.

Reply:
Good job in implementing the changes 👍